src/storage-controller/src/collection_mgmt.rs

```diff
@@ -1028,271 +1032,429 @@ where
     pub(crate) persist: Arc<PersistClientCache>,
 }
 
-/// Spawns an [`mz_ore::task`] that will continuously bump the upper for the specified collection,
-/// and append data that is sent via the provided [`mpsc::UnboundedSender`].
+/// A task that writes to an append only collection and continuously bumps the upper for the specified
+/// collection.
 ///
-/// TODO(parkmycar): One day if we want to customize the tick interval for each collection, that
-/// should be done here.
-/// TODO(parkmycar): Maybe add prometheus metrics for each collection?
-fn append_only_write_task<T>(
+/// For status history collections, this task can deduplicate redundant [`Status`]s.
```

Comment:
For things like this, there's also "[`Statuses`](Status)", for when you want to change the text of the link, btw

Reply:
Updated